weblate/checks/same.py

```diff
@@ -140,6 +140,14 @@ def test_word(word):
     return len(word) <= 2 or word in SAME_BLACKLIST or word in LANGUAGES
 
 
+def strip_placeholders(msg, unit):
+    regexp = re.compile(
+        "|".join(re.escape(param) for param in unit.all_flags.get_value("placeholders"))
+    )
+    stripped = regexp.sub("", msg)
+    return stripped
```

Comment:
Please simplify this:

* Pre-comiling regexp doesn't bring any benefits here as it is used just once. Better use directly `re.sub`.
* There is no need for local variable `stripped`, the value should be returned directly.